docs/usage/weather.rst

```diff
@@ -0,0 +1,118 @@
+Configuring Weather and Time
+============================
+
+Holodeck worlds have weather and time that can be configured, either with
+:ref:`scenarios` or programmatically in real time.
+
+See the :class:`~holodeck.weather.WeatherController` documentation for reference.
+
+Weather Options
+~~~~~~~~~~~~~~~
+
+Type
+----
+
+.. image:: weathers.png
+   :scale: 55%
+
+Holodeck worlds have three possible types of weather: **"sunny,"** **"cloudy,"** and **"rain."**
```

Comment:
Make sure to wrap at 79 chars (see docs/README.md for help on installing a linter.

Adding this to VSCode's settings.json is helpful:
```json
    "[restructuredtext]": {
        "editor.rulers": [79]
    },
```

Reply:
Also I would code format this if it's meant to be a literal, ie 
```reStructuredText
weather: ``sunny``, ``cloudy``,  and ``rain``.
```